generators/client/templates/angular/src/main/webapp/app/core/login/login.service.ts.ejs

```diff
@@ -83,8 +83,20 @@ export class LoginService {
             this.principal.authenticate(null);
         }
         <%_ } else { _%>
-        this.authServerProvider.logout().subscribe();
-        this.principal.authenticate(null);
+        this.authServerProvider.logout().subscribe(response => {
+            this.principal.authenticate(null);
+            const data = response.body;
+            let logoutUrl = data.logoutUrl;
+            // keycloak
+            if (logoutUrl.indexOf('localhost') > -1) {
```

Comment:
This test seems odd to me, Keycloak is not necessarily exposed on localhost, no?

Reply:
You are correct, and Okta is not necessarily exposed on oktapreview.com. I'm open to suggestions for better logic.